github/Repository.py

```diff
@@ -1393,6 +1393,28 @@ def __create_pull(self, **kwds):
             self._requester, headers, data, completed=True
         )
 
+    def create_transfer_request(
```

Comment:
This doesn't match the return type then? Either this creates a request to move the repository, which will happen at some point in the future, and the user then needs to go and use get_repo on the new name, or this should be called transfer and return the new repository object.